xbmc/settings/SubtitlesSettings.cpp

```diff
@@ -77,6 +77,11 @@ void CSubtitlesSettings::OnSettingChanged(const std::shared_ptr<const CSetting>&
 
   SetChanged();
   NotifyObservers(ObservableMessageSettingsChanged);
+  if (setting->GetId() == CSettings::SETTING_SUBTITLES_ALIGN)
+  {
+    SetChanged();
+    NotifyObservers(ObservableMessageSubtitleAlignChanged);
```

Comment:
Can you make `ObservableMessageSubtitleAlignChanged` a bit more generic i.e. not "scoped" to subtitles? Something like ObservableMessagePositioningChanged would do for me. Those messages are supposed to be generic so we need to somehow extend them if we want them scoped per component or make them generic enough to be reused by all